canairio/settings/testing.py

```diff
@@ -0,0 +1,19 @@
+from .base import *
+
+ENVIRONMENT = 'test'
+INFLUXDB_HOST = 'influxdb'
+try:
+    DATABASE_HOSTNAME = getenvvar('DATABASE_HOSTNAME')
+except:
+    DATABASE_HOSTNAME = 'timescaledb'
```

Comment:
Create a default kwarg for `getenvvar()`

Reply:
Fixed this one with the refactor to use more the base and the envvars